packages/core/database/lib/connection.js

```diff
@@ -17,39 +15,67 @@ const trySqlitePackage = (packageName) => {
     throw error;
   }
 };
+
+/**
+ * SQLite
+ */
+const SqliteClient = require('knex/lib/dialects/sqlite3/index');
+
 class LegacySqliteClient extends SqliteClient {
   _driver() {
     return require('sqlite3');
   }
 }
 
-const clientMap = {
-  'better-sqlite3': 'better-sqlite3',
-  '@vscode/sqlite3': 'sqlite',
-  sqlite3: LegacySqliteClient,
-};
-
 const getSqlitePackageName = () => {
-  // NOTE: allow forcing the package to use (mostly used for testing purposes)
```

Comment:
any reason to remove the NOTE here? that's pretty standard to outline specific things to keep in mind 🤔

Reply:
It seemed redundant since it's just a simple comment, but I'll add it back :)

Reply:
Yeah it's more to draw the attention version simple comments but it's not clear cut nevermind